models/activities/notification.go

```diff
@@ -71,6 +72,15 @@ type Notification struct {
 	UpdatedUnix timeutil.TimeStamp `xorm:"updated INDEX NOT NULL"`
 }
 
+// TableIndices implements xorm's TableIndices interface
+func (n *Notification) TableIndices() []*schemas.Index {
```

Comment:
Once you created the `TableIndices` method, please remove all tags on the struct for the xorm index.